sdk/storage/storage-internal-avro/src/utils/utils.common.ts

```diff
@@ -0,0 +1,26 @@
+export interface Metadata {
```

Comment:
This is `Record<string, string>`

Reply:
Changed to use `Record` and removed the `Metadata` interface.